torch_topological/nn/sliced_wasserstein_kernel.py

```diff
@@ -0,0 +1,73 @@
+"""Sliced Wasserstein kernel implementation."""
+
+
+import torch
+
+from torch_topological.nn import SlicedWassersteinDistance
+
+from torch_topological.utils import wrap_if_not_iterable
+
+
+class SlicedWassersteinKernel(torch.nn.Module):
+    """Calculate sliced Wasserstein kernel between persistence diagrams.
+
+    This is an implementation of the sliced Wasserstein kernel between
+    persistence diagrams, following [Carriere17a]_.
+
+    References
+    ----------
+    .. [Carriere17a] M. Carrière et al., "Sliced Wasserstein Kernel for
+    Persistence Diagrams", *Proceedings of the 34th International
+    Conference on Machine Learning*, PMLR 70, pp. 664--673, 2017.
```

Comment:
Please fix indent here:

```
.. [Carriere17a] M. Carrière et al., "Sliced Wasserstein Kernel for
   Persistence Diagrams", *Proceedings of the 34th International
   Conference on Machine Learning*, PMLR 70, pp. 664--673, 2017.
 ```
 
Else, this causes problems when creating the documentation.